### DISCUSSION TITLE
Windows frame not updateing

### DISCUSSION THREAD

**Discussion by khorapuppy** [emilk/egui#841] [Q&A]
For a computer with the following specs on windows 10, the GUI doesn't ever repaint the window.

Device name     DESKTOP-KFMJKS4
Processor       Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz   2.59 GHz
Installed RAM   16.0 GB (15.8 GB usable)
Device ID       38F0124F-1EED-456E-8408-F2B553362E8B
Product ID      00325-81470-14268-AAOEM
System type     64-bit operating system, x64-based processor
Pen and touch   No pen or touch input is available for this display

It's still capable of saving information on logout because when the program is closed and opened again, the information is updated. egui has also worked on every other Window's computer I tested the program on. Is there anything specific to the computer that egui has glitches on?

**Comment by emilk** [comment]
Can you please try it with the new `egui_glow` backend? Instructions at https://github.com/emilk/egui/tree/master/eframe#alternatives

**Reply by khorapuppy** [reply]
I will when I have access to the person who has the computer. It's taking a while for them to respond
